src/main/java/mate/jdbc/controller/CarAddController.java

```diff
@@ -21,10 +23,11 @@
 public class CarAddController extends HttpServlet {
     private final CarDao carDao = (CarDao) InjectorUtils.getInstance(CarDao.class);
```

Comment:
@EliosPride you don't need this anymore here

Reply:
@al-pavlenko 2